### PR TITLE
feat(4allportal): implement webdav event handler

### PR DESCRIPTION
**Pull request by sstepanenko-sc**

### CODE REVIEW

**Comment by mstroecker** [comment]
@tasches @sstepanenko-sc 
We have a new implementation design, so this goes back to draft. (See GitLab issues)

**Review by mstroecker — CHANGES_REQUESTED** [review]
As described in GitLab

File: charts/4allportal/templates/webdav/configmap.yaml [hunk 1]
```diff
@@ -124,4 +125,23 @@ data:
...
+        OP="Unknown"
+        if [[ "${arr[0]}" == "201" && "${arr[1]}" == "PUT" ]] then
+          OP="Create"
+        elif [[ "${arr[0]}" == "204" && "${arr[1]}" == "PUT" ]] then
+          OP="Update"
+        elif [[ "${arr[0]}" == "204" && "${arr[1]}" == "DELETE" ]] then
+          OP="Delete"
+        fi
+        echo "Send notification to $ENDPOINT: $OP ${arr[2]}" > /proc/self/fd/1
+        #wget -S ${ENDPOINT} 2>&1 | grep "HTTP/" | awk '{print $2}'
```
Comment (mstroecker) [inline charts/4allportal/templates/webdav/configmap.yaml:143]: I think wget is primarily a tool to download stuff and less commonly used. I'd prefer curl:

Sth. like this:
```bash
RES_CODE=$(curl -sw "%{http_code}\n" -o /dev/null ${ENDPOINT})
```

File: charts/base-cluster/README.md [hunk 2]
```diff
@@ -207,6 +207,9 @@ This helm chart requires flux v2 to be installed (https://fluxcd.io/docs/install
 | rbac.view.users | list | `[]` |  |
 | speedtest.enabled | bool | `true` |  |
 | speedtest.host | string | `"speedtest"` |  |
+| speedtest.image.registry | string | `""` |  |
+| speedtest.image.repository | string | `"adolfintel/speedtest"` |  |
+| speedtest.image.tag | string | `"latest"` |  |
```
Comment (mstroecker) [inline charts/base-cluster/README.md:212]: Something is wrong, this shouldn't be a change in here?

**Review by mstroecker — CHANGES_REQUESTED** [review]
You should recreate the MR with your changes, something got mixed up with main.

File: charts/4allportal/templates/webdav/configmap.yaml (hunk 1 above)
Comment (mstroecker) [reply charts/4allportal/templates/webdav/configmap.yaml:143]: Not sure if alpine is sufficient, but `quay.io/curl/curl` could be a good image.

Comment (sstepanenko-sc) [reply charts/4allportal/templates/webdav/configmap.yaml:143]: I've tried several images with 'curl' but none of them has 'bash' inside.
I agree, 'curl' is more flexible for processing HTTP, but 'wget' also should cover all our needs. It supports GET and POST requests and we can add auth header.

File: charts/base-cluster/README.md (hunk 2 above)
Comment (sstepanenko-sc) [reply charts/base-cluster/README.md:212]: Yesterday some parts mixed with main branch (and default commit message become incompatible on validation step) so I've tryed to restore it, but in same time extra changes (chore) applied on all branches.
Currently 'md' is in sync

**Pull request closed without merging**